test/gurka/test_avoids.cc

```diff
@@ -137,7 +137,7 @@ TEST_F(AvoidTest, TestMaxPolygonPerimeter) {
   // make sure the right exception is thrown
   try {
     gurka::do_action(Options::route, avoid_map, req);
-  } catch (const valhalla_exception_t& err) { EXPECT_EQ(err.code, 167); } catch (...) {
+  } catch (const valhalla_exception_t& err) { EXPECT_EQ(err.code, 167UL); } catch (...) {
```

Comment:
this is part of the other TODO: for some reason we're triggering now warnings in `gtest` with `--warning-sign-compare`. I started to change it but did only this file. do you have an idea @cvvergara ? I don't see how my changes can make the previously warning-free gurka files now trigger warnings..